hazelcast-integration/springboot-caching/.gitignore

```diff
@@ -0,0 +1,65 @@
+
```

Comment:
I think we do not introduce seperate `.gitignore` files with new samples. Is this neccessary?